ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/Main.java

```diff
@@ -0,0 +1,69 @@
+package edu.usb.argos.ASTProcessor;
+
+import edu.usb.argos.ASTProcessor.antlr.JavaLexer;
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.classes.JavaConstructorVisitor;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+
+public class Main {
```

Comment:
Please update your branch with the latest changes because this file has already been removed since it was added for quick testing.

Reply:
Done